docs/infrastructure/EKS/backend/variables.tf

```diff
@@ -2,12 +2,12 @@
 
 variable "project_tag" {
   description = "Project tag to add to s3 and dynamodb names"
-  type = string
-  default = "codacy"
+  type        = string
+  default     = "codacy"
 }
 
 variable "custom_tags" {
   description = "Map of custom tags to apply to every resource"
-  type = map(string)
-  default = null
+  type        = map(string)
+  default     = {}
```

Comment:
I think this was working well with `null` (cf. https://www.terraform.io/docs/configuration/expressions.html#null ). Did you run into some issues?

Reply:
Yep, got an error saying it cant be null :/

Reply:
These is the error:
![image](https://user-images.githubusercontent.com/3610528/82685793-b36e0180-9c4c-11ea-904d-84a4e78eb563.png)

Terraform versions:
![image](https://user-images.githubusercontent.com/3610528/82685824-bbc63c80-9c4c-11ea-99f7-cb7e03989acf.png)

Reply:
👍 I figured you wouldn't be fixing it if it wasn't broken, and was curious.